data-jpa/src/main/java/com/example/data/jpa/CLR.java

```diff
@@ -61,6 +66,18 @@ private void findById(List<Author> authors) {
 		System.out.printf("findById(): author2 = %s%n", author2);
 	}
 
+	private void entityGraph() {
+
+		Book book = new Book(null, "Spring in Action");
+		Author author = new Author(null, "Craig Walls", Collections.emptySet());
+		book.getAuthors().add(author);
+		bookRepository.save(book);
+
+		// Book loaded = bookRepository.findByTitleWithNamedGraph("Spring in Action");
```

Comment:
Is this commented out on purpose?

Reply:
yes - see comment below. I can move the entity graph feature to a new sample, but cannot test it within the same configuration.